helpers/html-elem-context-provider.js

```diff
@@ -0,0 +1,32 @@
+const attributeCallbacks = new Map();
+
+const contextObserver = new MutationObserver(mutations => {
+	mutations.forEach(mutation => {
+		const callbacks = attributeCallbacks.get(mutation.attributeName);
+		if (callbacks !== undefined && callbacks.length > 0) {
+			callbacks.forEach(callback => callback());
+		}
+	});
+});
+
+let observerInitialized = false;
+
+export function getContext(attributeName) {
+	const contextString = document.documentElement.hasAttribute(attributeName)
+		? document.documentElement.getAttribute(attributeName)
+		: undefined;
+
+	return JSON.parse(contextString);
```

Comment:
`JSON.parse(undefined)` throws... is that what we want or should it just return `undefined`?

Reply:
I guess in theory this could be any arbitrary string, which could throw. I can wrap it in a try-catch (though I'll do that in the BSI move).